data/hai/unfettered side missions.txt

```diff
@@ -1426,3 +1427,132 @@ mission "Unfettered: Ambush"
 	on enter
 		# SPOILER: he doesn't want them to get destroyed by the Hai because that would damage Alpha-Unfettered reputations, also he takes it as a challenge to his strength (except he's on your ship)
 		dialog `Upon entering the system you see multiple Cerberus ships, ready to attack you. Choot'k sends a message to the Hai. "Do not interfere, it would damage both my honor and <last>'s!" He did not ask for your opinion, but thankfully you have seen worse than a bunch of pirates.`
+
+
+
+mission "Unfettered: Hunger Stories 1"
+	name `Transport family to <destination>`
+	description `Transport a bereaved Unfettered family to <destination>.`
+	source "Firelode"
+	destination "Hai-home"
+	clearance `You contact the authorities when you enter the system and they say they will have a team waiting when you land.`
+	to offer
+		has "unfettered: hunger"
+	on offer
+		conversation
+			`You are walking past one of the Unfettered government offices in the spaceport when a woman walks out clearly distraught. Her clothes hang loose on her small frame. An infant clings tightly to her fur and a small boy walks behind her with a sombre expression. A collection of silver oak leaves is pinned to his jacket.`
+			choice
+				`	(Approach her.)`
+				`	(Get on with my business.)`
+					goto business
+
+			`	Your approach startles her out of her distraction and she pulls the boy close. When you speak to her it's clear she doesn't understand you. You look around for an Unfettered who could translate, but she grabs your sleeve and shakes her head. Eventually one of your crew who understands a little Hai gives it a try.`
+			`	"They have told me my husband is dead. The theiving raptors killed him. And now the boy has a medal but no father. They gave me his death money but there is no food and I don't know what we will do."`
+			`	You know that the Hai send food aid to the Unfettered and you've not seen anyone obviously hungry in Hai space. They would probably take this family in.`
+			choice
+				`	(Offer to take them to <destination>.)`
+				`	(Offer your condolences and move on.)`
+					goto "condolences"
```

Comment:
```suggestion
					goto condolences
```
dont need those commas when its in one word; but smth like "big word" would need them